deploy/olm/storageos/storageos.clusterserviceversion.yaml

```diff
@@ -358,16 +366,18 @@ spec:
                     fieldRef:
                       fieldPath: metadata.annotations['olm.targetNamespaces']
                 - name: OPERATOR_NAME
-                  value: cluster-operator
+                  value: storageos-cluster-operator
```

Comment:
Does this affect upgrades?  Do we need to test anything special?

Reply:
No, this shouldn't affect upgrades. It's just env var change. Renaming the deployment resource would have some affect on upgrades.

Reply:
This name change is required because the metrics Service uses operator's name as a label selector and it's not configurable. Setting it equal to the `name` label of the operator Deployment makes it work.